lib/config.js

```diff
@@ -8,7 +8,7 @@ var path = require('path');
 config['messages-client'] = config['messages-client'] || {};
 config['messages-client'].clients = config['messages-client'].clients || [];
 config['messages-client'].maxPollErrorsBeforeNotify =
-  config['messages-client'].maxPollErrorsBeforeNotify || 3;
+  ('maxPollErrorsBeforeNotify' in config['messages-client']) || 3;
```

Comment:
This needs to be a ternary ... right now it's going to set this equal to `true` or `3`. We want:

```js
('maxPollErrorsBeforeNotify' in config['messages-client'] ?
  config['messages-client']['maxPollErrorsBeforeNotify'] : 3);
```

Which is pretty verbosely gross but maybe some destructuring stuff from ES2015 will make that nicer in the future. Could create a var for config['messages-client'] if we wanted to make that a little nicer now.